Dockerfile.stream8

```diff
@@ -3,18 +3,22 @@ FROM quay.io/centos/centos:stream8
 # Install oVirt repositories
 RUN dnf copr enable -y ovirt/ovirt-master-snapshot centos-stream-8 \
     && dnf install -y ovirt-release-master
-# Configure CS8 repositories
-RUN dnf module enable -y pki-deps javapackages-tools
 
-# Install required packages
+# Configure CS8 modules
+RUN dnf module enable -y pki-deps javapackages-tools nodejs:14
+
+# Install packages required for oVirt github actions, common source ball generation
 RUN dnf install -y \
         createrepo_c \
-        dnf-utils
+        dnf-utils \
+        git \
```

Comment:
why adding these? they are supposed to get in via builddep below, and I beleive they do. What's missing?

Reply:
I'm not sure how `git` and `rpm-build` get installed, but they both are ultimately available in the build container.  So I take your point there.  They're not explicitly needed, but I like explicitly seeing them here.

`jq` is used by a few some `build.sh` scripts when creating the source archive and isn't used during any project builds so never shows up in a `BuildRequires` spec file line.

The bare minimum needed is to enable the nodejs:14 module before the rest of the `dnf install`s, and to install `jq`.

Reply:
:shrug: 
Ok so then the `BuildRequires` for a project is for both the spec's `%build` script and for the scripts that build the sources prior to dropping into the `rpmbuild`?

Reply:
I've updated this PR to only add enabling of `nodejs:14` module in CS8.